src/v2/System/Relay/SystemQueryRenderer.tsx

```diff
@@ -1,34 +1,52 @@
 import React from "react"
-// FIXME: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/37950
-// @ts-ignore
-import { QueryRenderer, QueryRendererProps } from "react-relay"
+import { useEffect } from "react"
+import { useState } from "react"
+import { QueryRenderer } from "react-relay"
 import { OperationType } from "relay-runtime"
 
-interface SystemQueryRendererState {
-  isMounted: boolean
+type QueryRendererProps = React.ComponentProps<typeof QueryRenderer>
```

Comment:
Like... this extracted type should be generic but I'm not sure how you'd be able to preserve it when extracting with `React.ComponentProps`